scipy/stats/tests/test_resampling.py

```diff
@@ -1523,35 +1523,6 @@ def statistic1d(x):
         assert res.statistic == res2.statistic
         assert_allclose(res.pvalue, res2.pvalue, atol=1e-2)
 
-    @pytest.mark.parametrize('alternative', ('less', 'greater'))
-    # Different conventions for two-sided p-value here VS ttest_ind.
-    # Eventually, we can add multiple options for the two-sided alternative
-    # here in permutation_test.
-    @pytest.mark.parametrize('permutations', (30, 1e9))
-    @pytest.mark.parametrize('axis', (0, 1, 2))
-    def test_against_permutation_ttest(self, alternative, permutations, axis):
```

Comment:
This tested `scipy.stats.permutation_test` against using the `permutations` argument of `ttest_ind`, but there are other, stronger tests now.